src/NuGet.Services.SearchService.Core/Program.cs

```diff
@@ -1,41 +1,21 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using Autofac.Extensions.DependencyInjection;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Configuration.EnvironmentVariables;
 using Microsoft.Extensions.Hosting;
-using System.Linq;
+using Microsoft.PackageManagement.Search.Web;
 
 namespace NuGet.Services.SearchService
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
+            HostBuilderHelper.CreateHostBuilder(args)
+               .ConfigureWebHostDefaults(webBuilder =>
+               {
+                   webBuilder.UseStartup<Startup>();
+               }).Build().Run();
```

Comment:
What do you think of using more new lines?

```suggestion
            HostBuilderHelper
                .CreateHostBuilder(args)
                .ConfigureWebHostDefaults(webBuilder =>
                {
                    webBuilder.UseStartup<Startup>();
                })
                .Build()
                .Run();
```

I don't feel strongly about this

Reply:
I think it looks nice